test/http-binding.spec.ts

```diff
@@ -43,4 +43,19 @@ describe('http-binding', () => {
         const result = await functionRunner(functionStub, [{ name: 'myResponse', type: 'http', direction: 'out' }]);
         expect(result).to.have.property('res', response);
     });
+    it('executes an Http Trigger with params', async () => {
+        const req = { params: { id: 'testId' } };
+        const functionStub = stub().resolves((context: Context) => context.bindingData);
```

Comment:
```suggestion
        const functionStub = stub().resolve();
```

The stub shouldn't resolve to a function.

Reply:
Thanks ! changed :)